roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-cnv-operator.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Create the CNV Operator namespace
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: openshift-cnv
+
+- name: Create CNV OperatorGroup kubevirt-hyperconverged-group
+  kubernetes.core.k8s:
+    state: present
+
+    definition:
+      apiVersion: operators.coreos.com/v1
+      kind: OperatorGroup
+      metadata:
+        name: kubevirt-hyperconverged-group
+        namespace: openshift-cnv
+      spec:
+        targetNamespaces:
+          - openshift-cnv
+
+- name: Create CNV Subscription
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: hco-operatorhub
+        namespace: openshift-cnv
+      spec:
+        source: redhat-operators
+        sourceNamespace: openshift-marketplace
+        name: kubevirt-hyperconverged
+
+- name: Wait
```

Comment:
You could add `wait` and `wait_condition` to the creation above. This way you could wait for the ready condition of the Subscription. IMO fixed time delays should be avoided.

Reply:
Removing unnecessary waits and using ansible k8s internal methods is something i'll look into for a later version of this code.